tooling/samples/index.js

```diff
@@ -40,8 +40,28 @@ const primitive = schema => {
 const sampleFromSchema = (schema, config = {}) => {
   const objectifySchema = objectify(schema);
   let { type } = objectifySchema;
-  const { example, properties, additionalProperties, items } = objectifySchema;
 
+  const hasPolymorphism = usesPolymorphism(objectifySchema);
+  if (hasPolymorphism) {
```

Comment:
I don't think you need this check, it seems to be covered by the following if.

Reply:
@Dashron, I added Line 45 in case there's `anyOf` or `oneOf` and there is a `type` present on the schema. The if on Line 46 will only check if there's an `allOf` and the else if will grab the rest!

Although, now that I'm typing this out, I'm wondering if I can add an `or` on the `(!type)` portion 🤔

Reply:
hey @Dashron, I updated this portion! GitHub isn't letting me respond to [this comment](https://github.com/readmeio/oas/pull/387/files/b23220047ef59525ab40a2061daf9211df70c0c5#discussion_r595424390), but I've refactored it so it handles schemas with `anyOf` or `oneOf` and a type.